### ISSUE TITLE
Training in a multi-label classification setting

### ISSUE THREAD

**Issue by guillaumejaume** [mahmoodlab/CLAM#34]
Hi, 

Thanks for open sourcing such a great project. I was wondering if the code could be used or easily modified for handling a multi-label classification tasks, eg in most of the prostate grading datasets, the objective is to predict both the primary and the secondary Gleason score leading to 2 different labels for each WSI. 

Thanks for the info!

**Comment by fedshyvana** [comment]
Yes, it should be fairly easy to modify. I don't do multi-label tasks, but intuitively you should be able to pass the groundtruth labels as a vector and modify the loss function to use a binary cross-entropy (with sigmoid activation) over the prediction score for each label class instead of a softmax activation?
